osu/map.py

```diff
@@ -52,9 +56,30 @@ def files(self):
 
     def zip(self):
         map_pack_file = MAP_PACKS_FOLDER.joinpath(f"{uuid.uuid4()}.zip")
-
+        print(map_pack_file)
         files = self.files()
         FileTools.zip_files(files, to_file=map_pack_file)
-        FileTools.delete_files(files)
 
         return map_pack_file
+
+    async def upload(self) -> str | None:
```

Comment:
here if we did want customisable path you can always do a path parameter that defaults to None if not give

it think its not needee but you can add if you think its useful

same with the zip function

Reply:
my main reply to this is *eh* cus realistically there should only be one upload function that uses this path to get there, so perhaps we keep it as a hard coded line for the time being?

Reply:
Upload now returns a string only, any errors are caught in map_pack.py and replies directly to the user who can raise issues with us